docs/getting-started/integration.md

```diff
@@ -15,87 +15,82 @@ Chart.js can be integrated with plain JavaScript or with different module loader
 
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
```

Comment:
is it weird that this refers to Chart.js 3? At the very least I think it should say "Chart.js 3+", but maybe we should change it to just "Chart.js"?

_Originally posted by @benmccann in https://github.com/chartjs/Chart.js/pull/9721#discussion_r938841542_